### PR TITLE
Inject CloudKMS Plugin container into Kube-APIServer pod.

### PR DESCRIPTION
**Pull request by immutableT**
**What this PR does / why we need it**:
Inject CloudKMS Plugin container into Kube-APIServer pod when etcd level encryption via CloudKMS Plugin is requested.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE

```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1715,6 +1731,67 @@ function start-kube-apiserver {
   sed -i -e "s@{{admission_controller_config_volume}}@${admission_controller_config_volume}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_mount}}@${image_policy_webhook_config_mount}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_volume}}@${image_policy_webhook_config_volume}@g" "${src_file}"
+
+  if [[ -z ${ETCD_KMS_KEY_ID:-} ]]; then
+    # Removing KMS related placeholders.
+    sed -i -e " {
```
Comment (mikedanese) [inline cluster/gce/gci/configure-helper.sh]: how does this multiline sed program work? does it execute all of these edits in one invocation?

File: cluster/gce/gci/configure-helper.sh [hunk 2]
```diff
@@ -1715,6 +1731,67 @@ function start-kube-apiserver {
...
+    } " "${src_file}"
+  else
+    local kms_plugin_src_file="${src_dir}/kms-plugin-container.manifest"
+
+    if [[ ! -f ${kms_plugin_src_file} ]]; then
+        echo "Error: KMS Integration was requested, but ${src_dir}/kms-plugin-container.manifest is missing."
+        exit 1
+    fi
+
+    if [[ ! -f ${encryption_provider_config_path} ]]; then
```
Comment (mikedanese) [inline cluster/gce/gci/configure-helper.sh]: wrap variable expansion in weak quotes i.e. `"${encryption_provider_config_path}"`

there are a couple other places in this cl that need this as well.

**Comment by immutableT** [comment]
@mikedanese 
I fixed indentations and wrapped variables in double quotes.

With respect to the syntax of sed where multiple commands are supplied between {}, I believe this is just syntactic sugar for avoiding typing sed all the time. In other words,
sed -i -e " {
      s@{{kms_socket_mount}}@@
      s@{{encryption_provider_mount}}@@
    } " "${src_file}"

is equivalent to 
sed -i -e  s@{{kms_socket_mount}}@@ "${src_file}"
sed -i -e  s@{{encryption_provider_mount}}@@ "${src_file}"

**Pull request merged**